pcstac/Dockerfile

```diff
@@ -5,37 +5,21 @@ RUN apt-get update && \
 
 ENV CURL_CA_BUNDLE /etc/ssl/certs/ca-certificates.crt
 
-RUN mkdir -p /opt/src/pcstac
-
-COPY pcstac/requirements.txt /opt/src/requirements.txt
-
 WORKDIR /opt/src
 
-RUN pip install -r requirements.txt
+COPY .isort.cfg .isort.cfg
+COPY pcstac/gunicorn_conf.py gunicorn_conf.py
 
-# This code used to depend on unreleased stac-fastapi
-# Including fix from https://github.com/stac-utils/stac-fastapi/pull/282
-# Based on 2.2.0 release
-RUN mkdir -p /opt/lib/stac-fastapi && \
-    cd /opt/lib/stac-fastapi && \
-    git init && \
-    git remote add origin https://github.com/stac-utils/stac-fastapi.git && \
-    git fetch origin a1e32cb3d32d3a7f031275296b98309b0c6be1cf && \
-    git reset --hard FETCH_HEAD && \
-    make pybase-install && \
-    make pgstac-install && \
-    make sqlalchemy-install
+COPY pcstac pcstac
+COPY pccommon pccommon
+RUN pip install ./pccommon ./pcstac["server"]
+RUN rm -rf pcstac pccommon
 
-
-COPY .isort.cfg /opt/src/.isort.cfg
-COPY pcstac /opt/src/pcstac
-COPY pccommon /opt/src/pccommon
+RUN pip install pypgstac==0.4.0
```

Comment:
I remove pypgstac from pcstac requirement because it's not used in the code itself